subprojects/uuid.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-git]
+url = https://github.com/util-linux/util-linux.git
+revision = eefff5aac7bce6979c950e3931a578efe03acbac
+
+[provide]
+libuuid = uuid_dep
```

Comment:
Shouldn't this be `uuid = uuid_dep` too?

Reply:
The specification for the provide section is that the left hand side (in this case "libuuid") is the name of the pkg-config config dependency it overrides.

Regardless of what the subproject used as the variable naming style, this would have never worked. That being said, it was always using the `fallback:` kwarg on versions of meson lower than 0.54

I would suggest in this case using

```ini
[provide]
dependency_names = uuid
```

Reply:
fixed